docs/source/services/key_value_stores/index.rst

```diff
@@ -0,0 +1,66 @@
+==================
+Key Value Services
+==================
+
+Overview
+========
+
+This matrix details the current state of available Key Value Services supported
+by the AtlasDB development staff.
+
++--------------------------------------+--------------------------------------+-------------------------------------------------------------+------------------------+------------------------------------------+----------------------------------------------+
+| Key Value Service                    | State                                | Service Versions Supported                                  | Recommended Versions   | Known Issues                             | Details                                      |
```

Comment:
I'd recommend using the "other" way of writing tables in rst, it looks way nicer.